### PR TITLE
Increase Margins

### PR DESCRIPTION
**Pull request by eksperimental**
- [x] Expand summary tables to the width of the page.
- [x] Increase margins in #content
- [x] And makes the top div cover 100% to the width

### CODE REVIEW

File: lib/ex_doc/formatter/html/templates/css/style.css [hunk 1]
```diff
@@ -70,6 +70,12 @@ body {
   line-height: 1.618;
 }
 
+#content {
+  padding: 0 28px 20px 28px;
+}
+
```
Comment (josevalim) [inline lib/ex_doc/formatter/html/templates/css/style.css]: Too many new lines? :D

File: lib/ex_doc/formatter/html/templates/css/style.css [hunk 2]
```diff
@@ -404,9 +410,9 @@ li.r2 {
 }
 
 div.breadcrumbs {
-  padding: 3px 6px;
+  padding: 3px 15px;
   background-color: #EEE;
-  margin: 0 -15px 10px;
+  margin: 0 -28px 10px;
```
Comment (milmazz) [inline lib/ex_doc/formatter/html/templates/css/style.css]: Why do you want to add more space? this change generate the following problem:

capture

Comment (eksperimental) [inline lib/ex_doc/formatter/html/templates/css/style.css]: yes, but in https://github.com/eksperimental/ex_doc/blob/fix2/lib/ex_doc/formatter/html/templates/css/style.css#L73-L75 I have increased the margin.
What broweser gave you that problem? Did you click on the breadcumb links for that to happen? if so, it's a bug that is unrelated to this change and it's happening even before these changes: https://github.com/elixir-lang/ex_doc/issues/254

File: lib/ex_doc/formatter/html/templates/css/style.css (hunk 1 above)
Comment (eksperimental) [inline lib/ex_doc/formatter/html/templates/css/style.css]: ooops.. :-P
fixed

File: lib/ex_doc/formatter/html/templates/css/style.css (hunk 2 above)
Comment (milmazz) [inline lib/ex_doc/formatter/html/templates/css/style.css]: Maybe was something in the cache, now I see it fine in Chrome. So, go ahead.
